packages/@sanity/desk-tool/src/components/BrokenReferences.js

```diff
@@ -22,50 +25,62 @@ function BrokenRefs(props) {
-  let missingText = 'do not exist'
-  let actionText = 'must be created'
-  if (unpublished.length > 0 && nonExistent.length > 0) {
-    missingText = 'do not exist and/or are not published'
-    actionText = 'must first be created and published'
-  } else if (unpublished.length > 0) {
-    missingText = `${isMultiDoc ? 'are' : 'is'} not published`
-    actionText = 'must first be published'
-  }
-
+  const renderNonExisting = nonExistent.length > 0
+  const renderUnpublished = !renderNonExisting
   return (
-    <DefaultPane title="Broken references">
-      <p>
-        The initial value for this document references {documentsText} that {missingText}. In order
-        to create <em>this</em> document, {actionDocText} {actionText}.
-      </p>
-
-      {unpublished.length > 0 && (
-        <>
-          <h2>Unpublished documents:</h2>
-          <ReferringDocumentsList
-            documents={unpublished.map(({id, type}) => ({_id: `drafts.${id}`, _type: type}))}
-          />
-        </>
-      )}
-
-      {nonExistent.length > 0 && (
-        <>
-          <h2>Non-existent document IDs:</h2>
-          <ul>
-            {nonExistent.map(doc => (
-              <li key={doc.id}>{doc.id}</li>
-            ))}
-          </ul>
-        </>
-      )}
+    <DefaultPane title={`New ${type}`} contentMaxWidth={672} isScrollable={false}>
+      <div className={styles.brokenReferences}>
+        {renderNonExisting && (
+          <PanePopover
+            icon
+            kind="error"
+            id="missing-references"
+            title="Missing references"
+            subtitle={`The new document can only reference existing documents. ${
+              nonExistent.length === 1 ? 'A document' : 'Documents'
+            } with the following ID${nonExistent.length === 1 ? ' was' : 's were'} not found:`}
+          >
+            <ul className={styles.tagList}>
+              {nonExistent.map(doc => (
+                <li className={styles.tagItem} key={doc.id}>
+                  {doc.id}
+                </li>
+              ))}
+            </ul>
+          </PanePopover>
+        )}
+        {renderUnpublished && (
+          <PanePopover
+            icon
+            kind="warning"
+            id="unpublished-documents"
+            title="Unpublished references"
+            subtitle={`A document can only refer to published documents. Publish the following ${
+              unpublished.length === 1 ? 'draft' : 'drafts'
+            } before creating
+            a new document.`}
+          >
+            <ReferringDocumentsList
+              documents={unpublished.map(({id, type, hasDraft}) => ({
+                _id: `drafts.${id}`,
+                _type: type,
+                _hasDraft: hasDraft
+              }))}
+            />
+          </PanePopover>
+        )}
+      </div>
+      <form className={styles.editor}>
+        <FormBuilder readOnly type={schemaType} schema={schema} />
+      </form>
```

Comment:
I need @bjoerge to approve this, I'm not sure whether there are side effects of doing something like this that we should consider.

Reply:
The only thing I can think of is focus handling and validation, etc. It will probably not be able to navigate through this form (e.g. open array items), but not sure if that's a problem here?

Reply:
I don't think any interaction with the form should be allowed here, seeing as the errors need to be resolved first before one can continue with the editing

Reply:
This should be fine, then 👍